source/developers/dotnet/quickstart.md

```diff
@@ -0,0 +1,212 @@
+
+// Iterate over the list of buckets.
+foreach (Bucket bucket in getListBucketsTask.Result.Buckets)
+{
+    Console.WriteLine(bucket.Name + " " + bucket.CreationDateDateTime);
+}
+
+```
+## Complete _File Uploader_ Example
+
+This example program connects to an object storage server, creates a bucket and uploads a file to the bucket.
+To run the following example, click on [Link] and start the project
+```cs
+using System;
+using Minio;
+using Minio.Exceptions;
+using Minio.DataModel;
+using System.Threading.Tasks;
+
+namespace FileUploader
+{
+    class FileUpload
+    {
+        static void Main(string[] args)
+        {
+            var endpoint  = "play.min.io";
+            var accessKey = "Q3AM3UQ867SPQQA43P2F";
+            var secretKey = "zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG";
+            try
+            {
+                var minio = new MinioClient()
+                                    .WithEndpoint(endpoint)
+                                    .WithCredentials(accessKey,
+                                             secretKey)
+                                    .WithSSL()
+                                    .Build();
+                FileUpload.Run(minio).Wait();
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+            Console.ReadLine();
+        }
+
+        // File uploader task.
+        private async static Task Run(MinioClient minio)
+        {
+            var bucketName = "mymusic";
+            var location   = "us-east-1";
+            var objectName = "golden-oldies.zip";
+            var filePath = "C:\\Users\\username\\Downloads\\golden_oldies.mp3";
+            var contentType = "application/zip";
+
+            try
+            {
+                // Make a bucket on the server, if not already present.
+                bool found = await minio.BucketExistsAsync(bucketName);
+                if (!found)
+                {
+                    await minio.MakeBucketAsync(bucketName, location);
+                }
+                // Upload a file to bucket.
+                await minio.PutObjectAsync(bucketName, objectName, filePath, contentType);
+                Console.WriteLine("Successfully uploaded " + objectName );
+            }
+            catch (MinioException e)
+            {
+                Console.WriteLine("File Upload Error: {0}", e.Message);
+            }
+        }
+    }
+}
+```
+
+## Running MinIO Client Examples
+#### On Windows
+* Clone this repository and open the Minio.Sln in Visual Studio 2017.
+
+* Enter your credentials and bucket name, object name etc.in Minio.Examples/Program.cs
+  Uncomment the example test cases such as below in Program.cs to run an example.
+```cs
+  //Cases.MakeBucket.Run(minioClient, bucketName).Wait();
+```
+* Run the Minio.Client.Examples project from Visual Studio
+#### On Linux
+
+##### Setting .NET SDK on Linux (Ubuntu 21.10)
```

Comment:
```suggestion
##### Setting .NET SDK on Linux (Ubuntu 21.10)
```

IT mentions here 21.10 however ...

Reply:
We'd need to fix this on the dotnet github repo - we just pull the markdown straight from there.

I need to do some fixes to the .NET markdown anyways, there are some linting errors that have been throwing issues here and there. Will look at these and those and open a PR for Ersan to look at.